pyproject.toml

```diff
@@ -5,12 +5,145 @@ requires = [
     "setuptools>=51.0.0",
     "wheel",
     "Cython>=0.29.32,<3",  # Note: sync with setup.py, environment.yml and asv.conf.json
-    "oldest-supported-numpy>=2022.8.16"
+    "oldest-supported-numpy>=2022.8.16",
+    "tomli; python_version < '3.11'" # Versioneer needs this
```

Comment:
You can leave this out. #49924 add `versioneer[toml]` to pyproject.toml

Reply:
I'll update after #49924 is merged.